README.md

```diff
@@ -3745,6 +3745,35 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='redundant-environment-key-implementation'></a>(<a href='#redundant-environment-key-implementation'>link</a>) **Prefer using the `@Entry` macro to define properties inside `EnvironmentValues`**. When adding properties to SwiftUI `EnvironemtnValues`, prefer using the compiler-synthesized property implementation when possible. [![SwiftFormat: environmentEntry](https://img.shields.io/badge/SwiftFormat-environmentEntry-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/develop/Rules.md#environmentEntry)
+
+    <details>
+
+    ### Why?
+
+    Manually-implemented environment keys are verbose, and keeping them up-to-date is error-prone.
```

Comment:
Just curious, which part of this do you think is error-prone about keeping these up-to-date? I do definitely agree that the old pattern is much more verbose.

Another thing to mention could be that manually implementing `EnvironmentKey` is a legacy pattern and `@Entry` was specifically intended to be a replacement.

Reply:
You are right, it may not be super error-prone. Let me update the details